rust/agama-lib/share/profile.schema.json

```diff
@@ -117,7 +129,8 @@
                     "type": "string"
                   },
                   "security": {
-                    "type": "string"
+                    "type": "string",
+                    "examples": ["? maybe this is actually an enum"]
```

Comment:
FIXME, need to look up in the code what is valid here

Reply:
@teclator can you advise please?
[service/test/agama/autoyast/wireless_reader_test.rb](https://github.com/openSUSE/agama/blob/master/service/test/agama/autoyast/wireless_reader_test.rb) and 
[web/src/components/network/WifiConnectionForm.jsx](https://github.com/openSUSE/agama/blob/master/web/src/components/network/WifiConnectionForm.jsx) seem to have different sets of values